### ISSUE TITLE
Events Page

### ISSUE THREAD

**Issue by iShubhamRana** [iShubhamRana/GDSC_NSUT_2022#3]

**Comment by Chaitanya134** [comment]
I'll do

**Comment by adnanhashmi09** [comment]
The design is on our figma.

**Comment by Chaitanya134** [comment]
Do you want me to implement the whole carousel on my own or use some existing libraries like owl-carousel?

**Comment by Chaitanya134** [comment]
Can you also confirm me about the font to use. In design I saw 'Inter' being used whereas in the css files Poppins and Barlow are imported

**Comment by adnanhashmi09** [comment]
> Do you want me to implement the whole carousel on my own or use some existing libraries like owl-carousel?

No, use existing libraries but avoid using jquery plugins. It will unnecessarily have a huge bundle size. You can use something like [slick.js](https://kenwheeler.github.io/slick/)

> Can you also confirm me about the font to use. In design I saw 'Inter' being used whereas in the css files Poppins and Barlow are imported

Use Inter and change wherever you find the inconsistencies. I will create a new issue for the font.

**Comment by Chaitanya134** [comment]
I'm done with the page only a bit of resposiveness is left. Do you want me to send a pull request now or after adding responsiveness?
